test/coverlet.collector/coverlet.collector.csproj

```diff
@@ -5,6 +5,7 @@
     <CheckEolTargetFramework>false</CheckEolTargetFramework>
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
     <IsTestProject>false</IsTestProject>
+    <LangVersion>preview</LangVersion>
```

Comment:
Not sure why but this project isn't inheriting the global settings.

Reply:
Probably because the global setting, was not preview before this project was in place and needed preview.